imports/api/courses.js

```diff
@@ -27,7 +27,8 @@ const coursePattern = {
   students: Match.Maybe([Helpers.MongoID]),
   instructors: Match.Maybe([Helpers.MongoID]),
   sessions: Match.Maybe([Helpers.MongoID]),
-  createdAt: Date
+  createdAt: Date,
+  requireVerified: Match.Maybe(Boolean)
```

Comment:
Is there a reason for this to be a "Maybe"? I would think we create courses with that set to true by default, and prof can set it to false if they want.